scss/app-ltr.scss

```diff
@@ -1,4 +1,6 @@
 @import "variables";
+@import "custom-tagify";
+@import "variables";
```

Comment:
` "variable"` is already imported in line 1. Does it have to be included here, again?